posts/inside-rust/2019-11-07-compiler-team-meeting.md

```diff
@@ -20,7 +20,7 @@ Rust 1.39 ships on Thursday!
 
 ### [wg-pgo](https://rust-lang.github.io/compiler-team/working-groups/pgo/)
 
-- PGO is available in the stable compiler. Docs are in the rustc-guide and the rustc-book
+- PGO is available in the stable compiler. Docs are in the rustc-dev-guide and the rustc-book
```

Comment:
Also lacks an active link

Reply:
addressed in 4367fad9d2bc955da039db41efc61627a0984c46